kura/examples/org.eclipse.kura.demo.modbus/src/main/java/org/eclipse/kura/demo/modbus/ModbusExample.java

```diff
@@ -223,26 +378,62 @@ private Properties getModbusProperties() {
                 if (bitsPerWord == null) {
                     bitsPerWord = "8";
                 }
-                if (ctopic == null) {
-                    ctopic = "/modbus/manager";
-                }
 
                 prop.setProperty("port", portName);
-                if (serialMode != null) {
-                    prop.setProperty("serialMode", serialMode);
-                }
                 prop.setProperty("exclusive", "false");
-                prop.setProperty("mode", "0");
-                prop.setProperty("slaveAddr", Slave);
                 prop.setProperty("baudRate", baudRate);
                 prop.setProperty("stopBits", stopBits);
                 prop.setProperty("parity", parity);
                 prop.setProperty("bitsPerWord", bitsPerWord);
-                prop.setProperty("controlTopic", ctopic);
-
             }
         }
+        prop.setProperty("mode", "0");
+        prop.setProperty("transmissionMode", "RTU");
+        prop.setProperty("respTimeout", "1000");
+        this.m_slaveAddr = Integer.valueOf(Slave);
+        
         return prop;
     }
 
+    // ----------------------------------------------------------------
+    //
+    // Cloud Application Callback Methods
+    //
+    // ----------------------------------------------------------------
+    @Override
+    public void onConnectionEstablished() {
+        // TODO Auto-generated method stub
```

Comment:
If we want not to implement these methods, remove the TODO comments.  This is a code refactoring standard. You can do it like this.
`public void onConnectionEstablished() {}`